config/config.exs

```diff
@@ -1,5 +1,5 @@
 use Mix.Config
 
-config :mavlink, dialect: APM
+config :mavlink, dialect: MAVLink.Dialect.APM.Types
```

Comment:
I'm not actually sure I'm supposed to include config.exs when it's packaged, these are for local testing in my working directory. My understanding is that when it's packaged the default env variables will be the ones in mix.exs application():

```
     env: [
        dialect: nil,
        system: 255,
        component: 250,
        connections: []
      ],
```
and users will override them with their own config.exs.

Reply:
Also the dialect would be MAVLink.Dialect.APM without the Types